homeassistant/components/energy/sensor.py

```diff
@@ -117,12 +118,13 @@ async def async_start(self) -> None:
 
     async def _process_manager_data(self) -> None:
         """Process manager data."""
-        to_add: list[SensorEntity] = []
+        to_add: list[EnergyCostSensor] = []
```

Comment:
The list can only contain EnergyCostSensors?

Reply:
No